### ISSUE TITLE
Missing CMake flag

### ISSUE THREAD

**Issue by sanguinariojoe** [SAP/fedem-solvers#9]
The fedem foundation would not compile if I do not add `set ( USE_VERTEXOBJ true )` at https://github.com/SAP/fedem-solvers/blob/main/CMakeLists.txt#L145

**Comment by kmoks** [comment]
I think this is the same issue that I discovered myself recently, and which is hopefully fixed in SAP/fedem-foundation#11. Can you verify that, @sanguinariojoe ? That is, included the commit https://github.com/SAP/fedem-foundation/pull/11/commits/b208764262675c5c7bb3470d34326e29b43d473e in your local clone and see if it then builds. The key is that the libraries `FFlVisualisation`and `FFlrLib` should be built only if `USE_VERTEXOBJ` is `true`. Thanks!

**Comment by sanguinariojoe** [comment]
Yeah, it seems to fix the issue!

```
# I am on fedem-solvers folder
git checkout CMakeLists.txt
cd fedem-foundation/
git remote add kmokstad https://github.com/kmokstad/fedem-foundation.git
git fetch kmokstad
git cherry-pick b208764
cd ../../fedem-solvers.build
make -j
# And.....
# It compiles!
```
